engine/interpreter/src/main/java/pl/touk/nussknacker/engine/definition/validator/adnotation/NotBlank.java

```diff
@@ -0,0 +1,14 @@
+package pl.touk.nussknacker.engine.definition.validator.adnotation;
```

Comment:
Why is it here? And not in API?

Reply:
please use standard annotation: https://docs.jboss.org/hibernate/beanvalidation/spec/2.0/api/javax/validation/constraints/NotBlank.html like we already used for NotNull

Reply:
You are right..

Reply:
This is annotation from hibernate javax validation.. Are you sure that we should use it?

Reply:
Changed